CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+# Change Log
+
+v0.5.0 **S3+CloudFront** (2020-07-05)
+
+- Added support for S3+CloudFront as a deployable target
+- Switched to a new YAML based configuration strategy
+- Deployments now copy new and changed files only
+
+v0.4.0 **Switched repository** (2020-05-24)
+
+Migrated the repository from Azure DevOps repos to GitHub.
```

Comment:
Nit-picky, should these items also be single-item bullet lists?

Reply:
What's our standard going to be if there's only one line. Do one-liners still get a lone bullet?

Reply:
We should decide on that.  I think a single bullet works simply for consistency, but for no other reason really.  In the future though, it would be nice that the release notes are generated using commit messages and the `CHANGELOG` auto-updated using that.  In that case, a single bullet makes sense as it would simply repeat one or more messages using the template we define.